assets/dev/js/editor/components/ai/hooks/use-user-info.js

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+import { getUserInformation } from '../api';
+
+const useUserInfo = () => {
```

Comment:
you already have an abstraction for all of it, why don't you use it here?

Reply:
It's not the same.